packages/search-ui-app-search-connector/search-ui-app-search-connector.d.ts

```diff
@@ -0,0 +1,65 @@
+declare module "@elastic/search-ui-app-search-connector" {
+  class AppSearchAPIConnector {
+    /**
+     * @typedef {Object.<string, *>} KeyValuePairs
+     */
+    /**
+     * @callback next
+     * @param {KeyValuePairs} updatedQueryOptions The options to send to the API
+     * @returns {Object} The response
```

Comment:
in typescript, there's no need for typing like `{Object}` in the comments, as those are a part of the types themselves

Reply:
@alexanderatallah These comments were copied over when running the `typescript` command to generate the definitions. I could disable that since the comments are included in the source files, but I thought maybe having them here as well would add some helpful context.

Reply:
And they are required in the source file, because a lot of the parameter typing in this is actually generated from the JSDoc in source.

Reply:
Ah ok!